src/Program.cs

```diff
@@ -142,5 +146,10 @@ private static void ConfigureServices(ServiceCollection serviceCollection, ICons
             serviceCollection.AddSingleton(new LoggerFactory().AddSimpleConsole(console, logLevel));
             serviceCollection.AddLogging();
         }
+        
+        private static string[] GetFileList(string files)

```

Comment:
If we make this method internal, it should be callable from the Test project. We could then test it for making directory separators uniform and for handling files that don't exist.

Reply:
Good idea!